package.json

```diff
@@ -29,6 +29,7 @@
     "generator-ibm-cloud-enablement": "0.0.109",
     "generator-ibm-service-enablement": "0.0.100",
     "handlebars": "^4.0.5",
+    "ibm-openapi-support": "^0.0.9",
```

Comment:
Looks like a merge error, remove one of the duplicates. Usually npm tries to keep these in alphabetical order, so probably removing the other one is best.